docs/guides/testing.md

```diff
@@ -77,30 +77,42 @@ const { create: actualCreate, createStore: actualCreateStore } =
 // a variable to hold reset functions for all stores declared in the app
 export const storeResetFns = new Set<() => void>()
 
+const createInternalFn = <T>(stateCreator: zustand.StateCreator<T>) => {
```

Comment:
Consider renaming `createInternalFn` and `createStoreInternalFn`
to `createTestStore`, `createStoreUncurried`, or similar.